### PR TITLE
Postman Study Group: Update Session#3 Materials

### PR DESCRIPTION
**Pull request by nora-weisser**
Updated information on Session3 within Postman Study Group:
- presentation
- materials and links
- additional information

## Type Of Change
- [x] Update documentation
- [ ] Add new event details
- [x] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

**Review by dricazenck — DISMISSED** [review]
🚀

File: resources/quality-guardians/Postman Study Group/Session #3/Session #3 - Introduction to Postman.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Session 3  - Introduction to Postman
+---
+### Description
+During this session, we will learn the essentials of Postman (API testing and development tool). Discover how Postman can streamline your workflow, automate tasks, and enhance your API testing capabilities.
```
Comment (dricazenck) [inline resources/quality-guardians/Postman Study Group/Session #3/Session #3 - Introduction to Postman.md:4]: Question, this happened already or will happen? Just to make sure about the sentence here

Comment (nora-weisser) [reply resources/quality-guardians/Postman Study Group/Session #3/Session #3 - Introduction to Postman.md:4]: Good point! 👍 I will rephrase a bit now!

Comment (nora-weisser) [inline resources/quality-guardians/Postman Study Group/Session #3/Session #3 - Introduction to Postman.md:4]:
```suggestion
Learn the essentials of Postman (API testing and development tool). Discover how Postman can streamline your workflow, automate tasks, and enhance your API testing capabilities.
```

**Review by dricazenck — APPROVED** [review]

**Pull request merged**